{{cookiecutter.folder_name}}/Makefile

```diff
@@ -36,12 +37,17 @@ tabdir=./tables
 tables=$(tabdir)/*.tex
 styles=./texmf/styles/*.{sty,cls}
 bibs=./texmf/bib/*.bst
-source=$(main).{tex,bbl,bib} lsstdesc.bib macros.tex authors.tex acknowledgments.tex
+source=$(main).{tex,bbl,bib} lsstdesc.bib authors.tex acknowledgments.tex
```

Comment:
People will see that it is included in `\bibliography{lsstdesc,main}`, and we can add some comments (both in `main.tex` and `main.bib`) hinting where the file is located. I think that according to the natural structure of the project it should be in `texmf/bib`.